src/app/facilities-and-resources/page.tsx

```diff
@@ -1,11 +1,221 @@
+            <div className="hidden lg:block absolute top-0 right-0">
+              <img src="/FacilitiesPage/tab.svg" alt="tab" />
+            </div>
+            <div className="flex justify-start lg:justify-center mt-0 mb-[20px] lg:items-center lg:p-[14px]">
+              <h2 className={`${alumniSans.className} font-bold text-7xl`}>
+                Servers
+              </h2>
+            </div>
+          </div>
+          <div className="order-4 w-full h-auto border border-white p-[14px] lg:col-span-4 row-start-8 row-span-5 border border-dashed p-[14px]">
+            <h2 className={`${alumniSans.className} font-bold text-4xl text-[#3F56E9] ${styles.boxshadow}`}>
+              03
+            </h2>
+            <h3 className={`${alumniSans.className} font-bold text-5xl ${styles.textstroke}`}>
+              Ithus Authentication Server
+            </h3>
+            <p>
+              Itus is an authentication server for systems in SSL. Each BTech
+              student of department has a username and password with which they
+              can login to the system. Itus uses LDAP for authentication.
+            </p>
+          </div>
+          <div className="order-5 w-full h-auto border border-white p-[14px] lg:col-span-4 row-start-8 row-span-5 border border-white p-[14px]">
+            <h2 className={`${alumniSans.className} font-bold text-4xl text-[#3F56E9] ${styles.boxshadow}`}>
+              04
+            </h2>
+            <h3 className={`${alumniSans.className} font-bold text-5xl ${styles.textstroke}`}>
+              Media Server
+            </h3>
+            <p>
+              Media Server is a warehouse of open source operating systems and
+              open courses from many prominent institutions of world.
+            </p>
+            <div className="py-[20px] mt-[14px]">
+              <Link href="http://192.168.40.130/" label="Media Server" />
+            </div>
+          </div>
+          <div className="order-6 w-full h-auto border border-white p-[14px] lg:col-span-4 row-start-8 row-span-5 border border-dashed border-white p-[14px]">
+            <h2 className={`${alumniSans.className} font-bold text-4xl text-[#3F56E9] ${styles.boxshadow}`}>
+              05
+            </h2>
+            <h3 className={`${alumniSans.className} font-bold text-5xl ${styles.textstroke}`}>
+              Mirror Server
+            </h3>
+            <p>
+              Mirror Server is the Ubuntu Mirror available internal to the
+              institution. It contains the up to date repository of supported
+              LTS version and recent short support versions
+            </p>
+          </div>
+        </div>
+      </section>
+      <section className="relative min-h-screen h-full flex flex-col py-24">
+        <SectionDivider />
+        <h2 className={`${alumniSans.className} font-bold text-7xl`}> Terminal Systems </h2>
```

Comment:
### Make the headings `text-5xl` on small screens (< lg) and `text-7xl` otherwise